field_manager/field_manager.F90

```diff
@@ -492,10 +499,11 @@ module field_manager_mod
   integer                                             :: max_index
   type (field_def), pointer                           :: first_field => NULL()
   type (field_def), pointer                           :: last_field => NULL()
-  integer, pointer, dimension(:)                      :: i_value => NULL()
```

Comment:
there's a pr open in main to change these to allocatables 
https://github.com/NOAA-GFDL/FMS/pull/1211/files